store/sqlstore/team_store.go

```diff
@@ -99,7 +99,6 @@ func (s SqlTeamStore) Update(team *model.Team) store.StoreChannel {
 			team.CreateAt = oldTeam.CreateAt
 			team.UpdateAt = model.GetMillis()
 			team.Name = oldTeam.Name
```

Comment:
I'm guessing this was hard-coded to carry the value forward at all times. I guess this means the name of the team is immutable too? (Just curious!)

Reply:
Yeah, I think so. This is silently forcing the team name not to change, instead of relying on database to error out (since name is set to be unique).